core/shared/src/test/scala/bobcats/HmacSuite.scala

```diff
@@ -29,6 +29,54 @@ class HmacSuite extends CatsEffectSuite {
 
   import HmacAlgorithm._
 
+  case class TestCase(key: ByteVector, data: ByteVector, digest: ByteVector)
+
+  // Test cases from RFC2022: https://datatracker.ietf.org/doc/html/rfc2202
+  val sha1TestCases = List(
+    TestCase(
+      ByteVector.fromHex("0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b").get,
```

Comment:
I think these can be written as `hex"0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b"` :)

Sadly, `ascii` is waiting for https://github.com/scodec/scodec-bits/pull/381